odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1304,6 +1304,15 @@ def _postprocess_tag_field(self, node, name_manager, node_info):
                     for arch, _view in self._get_x2many_missing_view_archs(field, node, node_info):
                         node.append(arch)
 
+                if (
+                    field.relational
```

Comment:
Why only relational fields? I guess it depends on the widget used, not on the type of the field.